enhancements/sandboxed-containers/sandboxed-containers-tech-preview.md

```diff
@@ -0,0 +1,620 @@
+                        description: FailedNodesList reflects the list of nodes that
+                          have failed kata operation
+                        items:
+                          description: FailedNodeStatus holds the name and the error
+                            message of the failed node
+                          properties:
+                            error:
+                              description: Error message of the failed node reported
+                                by the installation daemon
+                              type: string
+                            name:
+                              description: Name of the failed node
+                              type: string
+                          required:
+                          - error
+                          - name
+                          type: object
+                        type: array
+                    type: object
+                  inProgress:
+                    description: InProgress reflects the status of nodes that are
+                      in the process of kata uninstallation
+                    properties:
+                      binariesUninstallNodesList:
+                        items:
+                          type: string
+                        type: array
+                      inProgressNodesCount:
+                        type: integer
+                    type: object
+                type: object
+              upgradeStatus:
+                description: Upgradestatus reflects the status of the ongoing kata
+                  upgrade
+                type: object
+            required:
+            - kataImage
+            - runtimeClass
+            - totalNodesCount
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+```
+
+When deploying the sandboxed containers operator, the admin has the option
+to deploy it on all its worker nodes, or to a subset of the worker nodes.
+
+The former is achieved by:
+```yaml=
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: example-kataconfig
+```
+
+And the latter can be done by taking advantage of the PoolSelector, and specifying the matching labels, as:
+```yaml=
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: example-kataconfig
+spec:
+  kataConfigPoolSelector:
+    matchLabels:
+       custom-kata1: test 
+```
+
+Thanks to the Kubernetes Runtime Class [scheduling
+support](https://v1-16.docs.kubernetes.io/docs/concepts/containers/runtime-class/#scheduling)
+present since `v1.16.0`, we ensure that when a workload requires a
+specific `RuntimeClass`, it will be scheduled on nodes that
+supports the specific `RuntimeClass`.
+
+Finally, it's important to mention that altough there's a **strong**
```

Comment:
I would either incorporate anything here to the `Installation` section suggested below or drop this.